code/game/objects/items/weapons/melee/misc.dm

```diff
@@ -23,11 +23,12 @@
 
 /obj/item/weapon/melee/sabre
```

Comment:
I believe this is an attempt to nerf this weapon again after being denied twice.